City-of-Pori/pori-kada
# PORI-381 Fixed section main news block to work like front page news, …

…fixed hover styles

To test:

1. drush fra -y && drush cc all
2. Create a site section if you don't already have one. (/satakunnan-pelastuslaitos might be available if you have synced lately)
3. Add some images to section front page and set them as main news. The satakunnan pelastuslaitos section should have already two news promoted like this, TESTI1 and TESTI2
4. Make sure the main news items look the same as front page main news on the main site.
5. Make sure the hover on news items looks good.
